CHANGELOG.md

```diff
@@ -7,11 +7,12 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Bug fixes: any changes that fix issues or bug in the existing code
 - Dependencies: updates of dependency versions
 
-## [Unreleased] - yyyy-mm-dd
+## [Unreleased] - 2024-01-02
```

Comment:
This is a format example, the date will be added when creating a Release version by Code Owners, please revert it back